src/main/java/org/opendatakit/suitcase/ui/PullPanel.java

```diff
@@ -82,13 +122,35 @@ public PullPanel(IOPanel parent) {
         this.add(pullButtonPanel, gbc);
     }
 
+    private void buildSelectedTableIdsArea(SelectedTablesListPanel selectedTablesListPanel,GridBagConstraints gridBagConstraints){
+
+        gridBagConstraints.weighty = 1;
+        JPanel labelPanel = new JPanel(new GridLayout(1, 1));
+
+        labelPanel.setBorder(BorderFactory.createEmptyBorder(0,10,0,20));
+        labelPanel.add(new JLabel("Selected Table Ids for download"));
+
+        this.add(labelPanel,gridBagConstraints);
+
+        JScrollPane tableIdsScrollPane = new JScrollPane(selectedTablesListPanel,JScrollPane.VERTICAL_SCROLLBAR_ALWAYS,   //Make JScrollPane scroll only vertically
+                JScrollPane.HORIZONTAL_SCROLLBAR_NEVER);
```

Comment:
These 2 constants are actually in `ScrollPaneConstans`, which `JscrollPane` implements, reference the interface instead.